### PR TITLE
Release artifacts for release v1.3.0

### PR DESCRIPTION
**Pull request by kunhwiko**
Updates the release version of IAM controller to v1.3.0

### CODE REVIEW

File: helm/templates/deployment.yaml [hunk 1]
```diff
@@ -151,11 +151,11 @@ spec:
       hostNetwork: {{ .Values.deployment.hostNetwork }}
       dnsPolicy: {{ .Values.deployment.dnsPolicy }}
       volumes:
-      {{- if .Values.aws.credentials.secretName -}}
+      {{- if .Values.aws.credentials.secretName }}
         - name: {{ .Values.aws.credentials.secretName }}
           secret:
             secretName: {{ .Values.aws.credentials.secretName }}
-      {{ end -}}
+      {{- end }}
```
Comment (kunhwiko) [inline helm/templates/deployment.yaml:158]: Hmm, these seem to be recent code generator changes?

Comment (a-hilaly) [reply helm/templates/deployment.yaml:158]: Yesir!

**Review by a-hilaly — APPROVED** [review]
Thank you @kunhwiko !
/approve
/lgtm

**Pull request merged**